### ISSUE TITLE
Examplemod crash

### ISSUE THREAD

**Issue by Mewtwoy** [SirChipingston/Food-Ores#3]
![remove](https://user-images.githubusercontent.com/9406969/55672966-6798a700-5856-11e9-9d5c-491108a84d20.PNG)

The mod is crashing because it is named examplemod, the same as another one

**Comment by SirChipingston** [comment]
This is unfortunately not fixable (or if there is a fix I do not know of it). This comes down to you either deleting the enchanted golden apple crafting mod or deleting Food Ores.

**Comment by Mewtwoy** [comment]
> This is unfortunately not fixable (or if there is a fix I do not know of it). This comes down to you either deleting the enchanted golden apple crafting mod or deleting Food Ores.

Possible fix, needs testing: ExampleMod.class needs to have name changes

**Comment by SirChipingston** [comment]
Good point will change tag now and proceed to mess around with some fixes.
